src/sasjs-build/index.js

```diff
@@ -457,6 +474,190 @@ async function getDependencies(filePaths) {
   return dependenciesContent.join('\n')
 }
 
+export function getProgramList(fileContent) {
+  let fileHeader
+  try {
+    fileHeader = fileContent.split('/**')[1].split('**/')[0]
+  } catch (e) {
+    console.error(
+      chalk.redBright(
+        'File header parse error.\nPlease make sure your file header is in the correct format.'
```

Comment:
we should probably just ignore (and console log) any .sas files that don't have the header

Occasionally I throw in sas programs into folders without that header just to get things in quickly

Reply:
We now return an empty list if the program doesn't have the comment header block.